docs/chapters/helloworld/index.rst

```diff
@@ -10,13 +10,20 @@ As a starter, we invite you to try Jina's "Hello, World" - a simple demo of imag
     jina hello-world
 
 
-...or even easier for Docker users, *no any install required*, simply:
+or even easier for Docker users, *no install required*, simply for MacOS:
 
 
 .. highlight:: bash
 .. code-block:: bash
 
-    docker run -v "$(pwd)/j:/j" jinaai/jina:devel hello-world --workdir /j --logserver && open j/hello-world.html
+    docker run -v "$(pwd)/j:/j" jinaai/jina hello-world --workdir /j && open j/hello-world.html
```

Comment:
aren't they the same for mac and Linux?

Reply:
Linux should replace open to xdc-open to open a html file